cnf-certification-test/platform/suite.go

```diff
@@ -173,3 +179,28 @@ func testTainted(env *provider.TestEnvironment) {
 	gomega.Expect(taintedNodes).To(gomega.BeNil())
 	gomega.Expect(errNodes).To(gomega.BeNil())
 }
+
+func testIsRedHatRelease(env *provider.TestEnvironment) {
+	ginkgo.By("should report a proper Red Hat version")
+	failedContainers := []string{}
+	logrus.Infof("looping through %d containers", len(env.Containers))
+	for _, cut := range env.Containers {
+		ginkgo.By(fmt.Sprintf("%s(%s) is checked for Red Hat version", cut.Podname, cut.Data.Name))
+		baseImageTester := isredhat.NewBaseImageTester(common.DefaultTimeout, clientsholder.NewClientsHolder(), clientsholder.Context{
+			Namespace:     cut.Namespace,
+			Podname:       cut.Podname,
+			Containername: cut.Data.Name,
+		})
+
+		result, err := baseImageTester.TestContainerIsRedHatRelease()
+		if err != nil {
+			logrus.Debug("failed to collect release information from container: ")
+		}
+		if !result {
+			failedContainers = append(failedContainers, cut.Namespace+"/"+cut.Podname+"/"+cut.Data.Name)
+			tnf.ClaimFilePrintf("Container: %s/%s (ns: %s) has failed the RHEL release check", cut.Podname, cut.Data.Name, cut.Namespace)
```

Comment:
use StringShort() here and in other places to display the container

Reply:
Done.